firefox-ios/Client/Frontend/PasswordGenerator/PasswordGeneratorViewController.swift

```diff
@@ -11,7 +11,17 @@ import WebKit
 
 class PasswordGeneratorViewController: UIViewController, StoreSubscriber, Themeable, Notifiable {
     private enum UX {
-        static let containerPadding: CGFloat = 20
+        static let containerVerticalPadding: CGFloat = 20
+        static var containerHorizontalPadding: CGFloat {
+            if ((UIDevice.current.userInterfaceIdiom == .phone) && UIDevice.current.orientation.isLandscape) ||
+                ((UIDevice.current.userInterfaceIdiom == .pad) && UIDevice.current.orientation.isPortrait) {
+                return 90
+            }
+            if (UIDevice.current.userInterfaceIdiom == .pad) && UIDevice.current.orientation.isLandscape {
+                return 270
+            }
+            return containerVerticalPadding
+        }
```

Comment:
Having computed properties in a static UX enum is not really idiomatic for Swift or our codebase. I'd recommend creating variable names for the numbers so they're not magic numbers, and then creating a function that you can just pass in.